internal/usage/usage.go

```diff
@@ -206,7 +206,7 @@ dbName and collection are required only for built-in roles.`
 	NotificationType                          = "Type of alert notification. Valid values are DATADOG, EMAIL, FLOWDOCK, GROUP (Project), ORG, OPS_GENIE, PAGER_DUTY, SLACK, SMS, USER, or VICTOR_OPS."
 	NotificationUsername                      = "Name of the Atlas user to which to send notifications."
 	NotificationVictorOpsRoutingKey           = "Routing key associated with your Splunk On-Call account."
-	SnapshotID                                = "Unique identifier of the snapshot to restore. You must specify a snapshotId for automated restores."
+	SnapshotID                                = "Unique identifier of the snapshot."
```

Comment:
I think this may have been an oversight on reusing the constant for multiple commands, 

>to restore. You must specify a snapshotId for automated restores

Still applies to restore commands but on a quick look it doesn't for commands like `snapshots describe`, do you know which commands you wanted to fix this for as I'd like to tkeep the original description for restore but create a new const for things like that snapshot describe

Reply:
Yes it's in the description. I noticed the wrong description while exporting a snapshot to an s3 bucket.

Reply:
Is it ok if I push some changes to your branch? if not let's roll this change and I can address as a separate PR

Reply:
Sorry it took a while to get back to this, I pushed some changes on your branch to keep the original usage where applicable and the new simplified one for newer commands where it doesn't have the same meaning